app/src/main/java/com/it/hungvt/movieapp/screen/detail/DetailViewModel.java

```diff
@@ -0,0 +1,93 @@
+package com.it.hungvt.movieapp.screen.detail;
+
+import android.content.Context;
+import android.content.Intent;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.widget.Toast;
+import com.it.hungvt.movieapp.data.model.Movie;
+import com.it.hungvt.movieapp.BR;
+import com.it.hungvt.movieapp.screen.home.HomeActivity;
```

Comment:
ERROR: Unused import: com.it.hungvt.movieapp.screen.home.HomeActivity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports